external/webclient/webclient.c

```diff
@@ -1123,7 +1124,6 @@ static pthread_addr_t wget_base(void *arg)
 	int sockfd = -1;
 	int ret;
 	int buf_len, sndlen, len;
```

Comment:
do we need 'buf_len' variable?

Reply:
It is used when sending, but not when receiving. It's corrected.